src/foundry/common/grid/hexagonal.d.mts

```diff
@@ -8,6 +8,13 @@ declare class HexagonalGrid extends BaseGrid {
    */
   constructor(config: HexagonalGrid.Configuration);
 
+  // declared here for type-checking, but values are limited to these in constructor
+  override type:
+    | typeof CONST.GRID_TYPES.HEXEVENQ
```

Comment:
This looks almost more like a keyboard smash than a real enum value haha.

I get what it is after looking at it for a second of course.